src/Queue/PageviewQueue.php

```
@@ -0,0 +1,114 @@
+use LukaLtaApi\Value\Device;
+use LukaLtaApi\Value\GeoLocation;
+use LukaLtaApi\Value\UserAgent;
+use LukaLtaApi\Value\WebTracking\Tracking\PageViewData;
+use LukaLtaApi\Value\WebTracking\Tracking\PageViewEvent;
+use LukaLtaApi\Value\WebTracking\Tracking\TrackingBatch;
+use LukaLtaApi\Value\WebTracking\Tracking\UrlParameter;
+
+class PageviewQueue
+{
+    private array $queue = [];
+    private int $interval = 10; // seconds
```

PHPMD (2.15.0)

Avoid unused private fields such as '$interval'. (undefined)

(UnusedPrivateField)